brewtils/rest/easy_client.py

```diff
@@ -50,6 +51,8 @@ def handle_response_failure(response, default_exc=RestError, raise_404=True):
         raise WaitExceededError(response.json())
     elif response.status_code == 409:
         raise ConflictError(response.json())
+    elif response.status_code == 413:
+        raise TooLargeError(response.json())
```

Comment:
Don't forget to put this in the comment block for the function under the ```Raises:``` section.

Reply:
:+1: